code/modules/power/gravitygenerator.dm

```diff
@@ -215,6 +215,19 @@ GLOBAL_LIST_EMPTY(gravity_generators)
 
 // Interaction
 
+/obj/machinery/gravity_generator/main/examine(mob/user)
+	. = ..()
+	if(machine_stat & BROKEN)
+		switch(broken_state)
```

Comment:
```suggestion
	if(!(machine_stat & BROKEN))
		return
	switch(broken_state)
```
Really small change but exit out of a proc early if possible rather than indenting the entire body of code inside an if statement.